jest.config.js

```diff
@@ -9,6 +9,9 @@ module.exports = {
     'jsx',
     'js'
   ],
+  setupFiles: [
+    '<rootDir>/spec/javascripts/__mocks__/global-mocks.js'
```

Comment:
We should move the `__mocks__` inside of `spec/javascripts` no? Another PR?

Reply:
Are they no already inside that folder? WDYM?